client/src/components/QA/QACardAnswers.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import Modal from 'react-modal';
+import axios from 'axios';
+import QuestionsAndAnswers from './QuestionsAndAnswers.jsx';
+
+// Modal.setAppElement('#app');
+if (process.env.NODE_ENV !== 'test') {
+  Modal.setAppElement('#app');
+}
+
+const QACardAnswers = ({ currentAnswers }) => {
+
+  // array for all answers for current question
+  const [answers, setAnswers] = useState([]);
+  const [answersLength, setAnswersLength] = useState(0);
+  // array for answers to render
+  const [answersToShow, setAnswersToShow] = useState([]);
+  const [answersToShowLength, setAnswersToShowLength] = useState(2);
+  // toggle is to make sure we avoid infinite rerender loop
+  const [toggle, setToggle] = useState(false);
+
+  useEffect(() => {
+    if (answers) {
+      for (var key in currentAnswers) {
+        answers.push(currentAnswers[key]);
+      }
+      // sets answers array
```

Comment:
This level of comment is probably closer to pseudocode and might not be necessary for those that understand the React hooks pattern.